src/Collector/VariableCollector.php

```diff
@@ -74,28 +73,38 @@ public function processNode(Node $node, Scope $scope): ?array
         }
 
         if ($nameNode instanceof Expr) {
-            return null;
+            $variableName = null;
```

Comment:
I need to catch assignments that cannot be evaluted but can have annotation for example `$this->template->{$varName}` can be annotated to allow analysis